docker-compose.yml

```diff
@@ -24,17 +24,11 @@ services:
       - ./docker/rskj/node.conf:/etc/rsk/node.conf
       - ./docker/rskj/data:/var/lib/rsk/.rsk
   
-  dev-liquidity-token-watcher:
-    image: "matterlabs/dev-liquidity-token-watcher:latest"
+  dev-ticker:
+    build:
+      dockerfile: ./docker/dev-ticker/Dockerfile
```

Comment:
```suggestion
      context: ./docker/dev-ticker/Dockerfile
```

Reply:
why?

Reply:
It looks the context is required, but we cannot get rid of `dockerfile` because we want to use `.` as context and `./docker/dev-ticker/Dockerfile` as docker file.